catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/AttributeImpl.java

```diff
@@ -146,6 +137,37 @@ public void clearValues() {
         values.clear();
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+
+        if (!(o instanceof Attribute)) {
```

Comment:
Not needed. `o instanceof Attribute` will be `false` if `o` is `null`. I'd like to take credit but this code was generated by IntelliJ :-)